### PR TITLE
remove offensive language in package ban config

### PR DESCRIPTION
**Pull request by biphasic**
I understand that some users do silly things with their system configuration and this ban is supposed to prevent that. I just improved the wording for any user who like me stumbles across this file on their system...

## The problem

...

## Solution

...

## PR Status

...

## How to test

...

### CODE REVIEW

**Review by zamentur — APPROVED** [review]
LGTM

File: data/hooks/conf_regen/10-apt [hunk 1]
```diff
@@ -17,19 +17,16 @@ Pin-Priority: -1" >> "${pending_dir}/etc/apt/preferences.d/extra_php_version"
...
-# So, *NO*
-# DO NOT do this.
-# DO NOT remove these lines.
-#
-# I warned you. I WARNED YOU! But did you listen to me?
-# Oooooh, noooo. You knew it all, didn't you?
+
+# PLEASE READ THIS WARNING AND DON'T EDIT THIS FILE
+
+# You are probably reading this file cause you tryed to install apache2 or
```
Comment (cyxae) [inline data/hooks/conf_regen/10-apt:23]: `because you tried` c’est mieux je pense que `cause you tryed`

File: data/hooks/conf_regen/10-apt [hunk 2]
```diff
@@ -17,19 +17,16 @@ Pin-Priority: -1" >> "${pending_dir}/etc/apt/preferences.d/extra_php_version"
...
-# DO NOT do this.
-# DO NOT remove these lines.
-#
-# I warned you. I WARNED YOU! But did you listen to me?
-# Oooooh, noooo. You knew it all, didn't you?
+
+# PLEASE READ THIS WARNING AND DON'T EDIT THIS FILE
+
+# You are probably reading this file cause you tryed to install apache2 or 
+# bind9. This 2 packages conflict with YunoHost.
```
Comment (cyxae) [inline data/hooks/conf_regen/10-apt:24]: `These` à la place de `this`

**Review by cyxae — CHANGES_REQUESTED** [review]
Quelques erreurs d’anglais mais c’est sûr que c’est moins violent comme ça :)

File: data/hooks/conf_regen/10-apt (hunk 1 above)
Comment (biphasic) [reply data/hooks/conf_regen/10-apt]: done

**Comment by biphasic** [comment]
Hello, I fixed the mentioned spelling mistakes

**Pull request merged**